packages/react/src/primitives/types/expander.ts

```diff
@@ -0,0 +1,42 @@
+import { ViewProps } from '../types/view';
+
+type ExpanderType = 'single' | 'multiple';
+
+export interface ExpanderProps extends ViewProps {
+  /**
+   * The value of the item(s) to expand.  Use on uncontrolled component.
+   */
+  defaultValue?: string | string[];
+
+  /**
+   * The controlled value of the item(s) to expand. Must be used in conjunction with onChange.
+   */
+  value?: string | string[];
+
+  /**
+   * Determines whether the opened item can be collapsed if this is a single expander.
```

Comment:
Only applies to `type`single
```suggestion
   * Determines whether an opened item can be collapsed
   * Note: only applies when `type` is `single`
```